Cockpit 291 (#629)
cockpit-project/cockpit-project.github.io: _posts/2023-05-03-cockpit-291.md

---
title: Cockpit 291
author: garrett
date: '2023-05-03'

> tags: cockpit, machines

Author: ```suggestion
tags: cockpit, machines, podman
```

slug: cockpit-291
category: release

> summary: ''

Author: ```suggestion
summary: 'PatternFly 5, create VMs with raw disks, VM CPU editing refinements'
```

---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from cockpit 291, cockpit-machines 289.1, and cockpit-podman 68:

## Update to patternfly 5 alpha

Cockpit is now an "early adopter" of PatternFly 5. Everything should look the same. We ensured all functional and visual tests pass. While this should not affect people using Cockpit, it definitely will affect everyone developing for Cockpit. Read more on [our PatternFly 5 blog post](https://cockpit-project.org/blog/patternfly-v5-alpha.html).

## Option to use RAW volume during VM creation

> Users can now choose RAW as the storage format for the primary disk of their VM during the VM creation process.

Author: "RAW" is not an acronym. It should be "raw". Also: We should not refer to people as "users". I'll reword this section.

![Screenshot from 2023-05-03 12-32-48]({{ site.baseurl }}/images/291-option-to-use-raw-volume-during-vm-creation.png)

## Redesign VM's CPU configuration

The CPU model dialog and vCPU topology have been combined into a single field and a dialog. Additionally, a few minor improvements have been added, including vCPU maximum and vCPU count now being number inputs. Furthermore, the dialog layout has been updated to a vertical form.

![Screenshot from 2023-05-03 12-32-03]({{ site.baseurl }}/images/291-redesign-vm-s-cpu-configuration.png)

> ![Screenshot from 2023-05-03 12-40-15]({{ site.baseurl }}/images/291-redesign-vm-s-cpu-configuration-2.png)

Author: People writing the descriptions need to have a blank line between images. I usually fix this myself.

> ![Screenshot from 2023-05-03 12-40-15]({{ site.baseurl }}/images/291-redesign-vm-s-cpu-configuration-2.png)

Author: ```suggestion

![Screenshot from 2023-05-03 12-40-15]({{ site.baseurl }}/images/291-redesign-vm-s-cpu-configuration-2.png)
```

---
title: Cockpit 291
author: mvollmer
date: '2023-05-03'
tags: cockpit, machines, podman
slug: cockpit-291
category: release
summary: 'PatternFly 5, create VMs with raw disks, VM CPU editing refinements'
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from cockpit 291, cockpit-machines 289.1, and cockpit-podman 68:

> ## Update to patternfly 5 alpha
> 
> Cockpit is now an "early adopter" of PatternFly 5. Everything should look the same. We ensured all functional and visual tests pass. While this should not affect people using Cockpit, it definitely will affect everyone developing for Cockpit. Read more on [our PatternFly 5 blog post](https://cockpit-project.org/blog/patternfly-v5-alpha.html).

Author: ```suggestion
## Update to PatternFly 5 alpha

Cockpit is now an "early adopter" of PatternFly 5. Everything should look the same. We ensured all functional and visual tests pass.

While this should not affect anyone using Cockpit, it definitely will affect everyone developing for Cockpit. Read more on [our PatternFly 5 blog post](https://cockpit-project.org/blog/patternfly-v5-alpha.html).
```

> ## Machines: Option to use RAW volume during VM creation
> 
> Users can now choose RAW as the storage format for the primary disk during the VM creation process.

Author: ```suggestion
## Machines: VM creation supports raw disk format

During virtual machine creation, there is now a raw disk option in the storage format selection.
```

![Screenshot from 2023-05-03 12-32-48]({{ site.baseurl }}/images/291-option-to-use-raw-volume-during-vm-creation.png)

> ## Machines: Redesign VM's CPU configuration
> 
> The CPU model dialog and vCPU topology have been combined into a single field and a dialog. Additionally, a few minor improvements have been added, including vCPU maximum and vCPU count now being number inputs. Furthermore, the dialog layout has been updated to a vertical form.

Author: ```suggestion
## Machines: Redesign virtual machine CPU configuration

The CPU model dialog and vCPU topology have been merged into a single overview entry and dialog. Additionally, a few minor improvements have been added, including vCPU maximum and vCPU count now being number inputs.
```
